components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.v1/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/v1/core/ServerTenantManagementService.java

```diff
@@ -45,6 +45,7 @@
 import org.wso2.carbon.stratos.common.exception.TenantManagementServerException;
 import org.wso2.carbon.stratos.common.exception.TenantMgtException;
 import org.wso2.carbon.stratos.common.util.ClaimsMgtUtil;
+import org.wso2.carbon.tenant.mgt.services.TenantMgtImpl;
```

Comment:
Please double check this import. Smells like cyclic dependency on imp usage.

We can reconsider the constant location TenantMgtImpl.DOMAIN_NAME or somewhere else depending on the above decision.

Reply:
Moved to https://github.com/wso2/carbon-multitenancy/pull/279